beacon-chain/types/attestation_test.go

```diff
@@ -56,3 +56,19 @@ func TestAttestation(t *testing.T) {
 		t.Errorf("verify attestation failed: %v", err)
 	}
 }
+
+func TestContainsValidator(t *testing.T) {
```

Comment:
This test essentially covers the same logic that used to be covered in `TestContainsAttestations`. `ContainsAttestation` was split into `getAttestation` and `attestation.ContainsValidator`. By splitting the function into two and testing those individual functions, the tests become a lot simpler.